src/renderers/WebGLRenderer.js

```diff
@@ -2901,7 +2905,7 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	}
 
-	function setCubeTexture ( texture, slot ) {
+	function setTextureCube ( texture, slot ) {
```

Comment:
Aww, sorry! Actually, just meant to fill in the history why the code was like you found it.

> Well, the function names didn't really reflect that difference and there is no documentation about this.

And this is true too :).